static/app/components/feedback/feedbackSearch.tsx

```diff
@@ -73,11 +108,21 @@ export default function FeedbackSearch({className, style}: Props) {
   const projectIds = usePageFilters().selection.projects;
   const {pathname, query} = useLocation();
   const organization = useOrganization();
-  const tags = useTags();
+  const organizationTags = useTags();
```

Comment:
And same comment about this again, that this can be fetched from the issue platform dataset instead

Reply:
Actually, there is one additional problem with using `useTags()` on feedback, which is that the request to fetch them isn't being made if you refresh on the feedback page. It looks like tags never load 😕 . But that's a problem in the existing implementation already

Reply:
Oh, is it because we don't call loadOrganizationTags like in replays?

Reply:
Yeah that must be the missing call

Reply:
So we could use `useFetchIssueTags` instead right? Which directly queries issue platform instead of the store

Reply:
Thanks for pointing this out! I'm thinking we do it in a follow-up, so this PR only adds the new component with no other changes

Reply:
You will want to use `useFetchOrganizationTags` directly since you only need the issue platform dataset. `useFetchIssueTags` also fetches tags from errors which you don't need here

Reply:
I see, thanks!!